Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.MigrationsTool/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace MvvmSeed.MigrationsTool
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //NOTES: We need this console projects to generate migrations since EF Tools are not available for the other platforms yet
+            //TODO: Do something cool here to generate migrations automatically
+            Console.WriteLine("WIP");
```

Comment:
I don't think so.. Right now I'm generating migrations manually when required, I just want to add some automation wherever's possible. Any ideas are welcome.